articles/protocols/saml/adfs.md

```diff
@@ -0,0 +1,175 @@
+---
+description: How to Set up an Auth0 SAML Connection Against ADFS
+toc: true
+---
+
+# ADFS SAML Connection
+
+Auth0 allows you to create a custom SAML connection to Microsoft's Active Directory Federation Services. In addition to getting a bit more flexibility when configuring your mappings, the SAML Connection allows you identity provider-initiated flows (this is something that you cannot do with WS-Fed).
+
+## Step 1: Configure ADFS
+
+To get your ADFS server ready, you'll need to:
+
+1. Add a relying party trust
+2. Edit your claim issuance policy
+3. Obtain your federation metadata
+4. Export the signing certificate to upload to Auth0 
+
+### Add a Relying Party Trust
+
+Begin by launching your instance of ADFS. Start the **Add Relying Party Trust** wizard.
+
+![](/media/articles/protocols/saml-adfs/saml1.png)
+
+On the **Welcome** page, choose **Claims aware** and click **Start**.
+
+![](/media/articles/protocols/saml-adfs/saml2.png)
```

Comment:
Could you put a [border](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#borders) on the wizard screenshots? Since this is not a full browser window screenshot, understanding where the text begins and where the screenshots ends can be a bit confusing.

![image](https://user-images.githubusercontent.com/11715799/29452254-55b46038-840e-11e7-90a2-41612b95fcee.png)